nuclear/module/NUClearModule.cmake

```diff
@@ -87,24 +98,35 @@ FUNCTION(NUCLEAR_MODULE)
                 DEPENDS ${python_file}
                 COMMENT "Copying updated python file ${python_file}"
             )
-
         ENDFOREACH(python_file)
 
         FILE(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src")
 
+        SET(ENV{PYTHONPATH} "${PYTHONPATH}:${PYTHON_SITE_PACKAGES}")
+        SET(ENV{NUCLEAR_MODULE_DIR} "${PROJECT_SOURCE_DIR}/${NUCLEAR_MODULE_DIR}")
+        SET(ENV{LD_LIBRARY_PATH} "/nubots/toolchain/${PLATFORM}/lib:/nubots/toolchain/lib")
```

Comment:
This has to be smarter than this, This code is shared between all NUClear projects

Reply:
Hmmm .... this will take some thought